docs/operations/api-reference.md

```diff
@@ -114,6 +114,44 @@ Returns the number of segments to load and drop, as well as the total segment lo
 
 Returns the serialized JSON of segments to load and drop for each Historical process.
 
+
+#### Segment Loading by Datasource
+
+You can verify if segments created by a recent ingestion task are loaded onto historicals and available for querying using the following APIs.
+An example workflow for this is:
```

Comment:
This example workflow seems useful, but also out of place given the rest of this document, which strictly documents API requests and responses instead of illustrating an example application of how to use the API. I'm not sure where else it should live, or if it should just be removed, or if maybe it is fine, but I think it is worth discussing.

Reply:
I think this example is useful. Especially when user is trying to verify if segments are loaded and available right after ingestion task is done. The workflow also point out the difference with the coordinator loadstatus API (and how they have different use case). I think the https://druid.apache.org/docs/latest/ingestion/faq.html might be a better place for this.